content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -210,9 +210,8 @@ Removed experiments: exp-bb09c
 
 ## Removing queued experiments
 
-When you've created experiments to be run in the queue with
-`dvc exp run --queue` and later decide not to run them, you can remove them with
-`dvc exp remove --queue`.
+When you've queued experiments with `dvc exp run --queue` and later decide not
+to run them, you can remove them with `dvc exp remove --queue`.
```

Comment:
Maybe mention both `dvc exp remove` and `dvc queue remove` or keep it generic as suggested above. However, whatever we decide, it should be consistent with the examples below (see line 236 below).

Reply:
Rolled back line 236.

> keep it generic as suggested above

That seems like a bigger task that applies to many places probably. I'll leave it up to you to create an issue, etc. if needed!